### PR TITLE
Migrated CommunityBridge from layer5 to layer5-ng

### LINKED ISSUE #707: [NG] Migrate site data for CommunityBridge to the layer5-ng site
**Description**
Currently, the internship page dedicated to [CommunityBridge](https://layer5ng.netlify.app/careers/programs/communitybridge/2020) on the [layer5-ng site](https://layer5ng.netlify.app/) points to a 404 error. 

**Expected Behavior**

The data from the [CommunityBridge](https://layer5.io/programs/communitybridge) on the [layer5.io site](https://layer5.io/) needs to be migrated to the layer5-ng site. 

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by geraldmaboshe**
Signed-off-by: Gerald Maboshe <maboshegerald1@gmail.com>

**Description**

This PR fixes #707 

**Notes for Reviewers**
N/A

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by geraldmaboshe** [comment]
I need help on how to go about migrating Other Projects and Mailing List sections. @chandrashritii @Tanuj22

**Comment by Tanuj22** [comment]
@geraldmaboshe You can ignore that. It is not needed currently

File: src/collections/programs/communitybridge/index.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+import { ProgramsWrapper } from "../Programs.style.js"
+import Button from "../../../reusecore/Button";
+import communitybridge_logo from "./communitybridge-blue.png";
+
+<ProgramsWrapper>
+
+
+  <img src={communitybridge_logo}/>
+  <h1>CommunityBridge</h1>
```
Comment (Tanuj22) [inline src/collections/programs/communitybridge/index.mdx:16]: Remove this

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Looking good! Just one change and we are good to go

**Comment by Tanuj22** [comment]
@geraldmaboshe I meant only remove the text :sweat_smile:

**Comment by geraldmaboshe** [comment]
> @geraldmaboshe I meant only remove the text 😅

You highlighted image too, lol. On it

File: src/collections/programs/communitybridge/index.mdx (hunk 1 above)
Comment (geraldmaboshe) [reply src/collections/programs/communitybridge/index.mdx]: > @geraldmaboshe I meant only remove the text 😅

Does the logo remain right aligned?

Comment (Tanuj22) [reply src/collections/programs/communitybridge/index.mdx]: Anywhere it looks good, works

**Review by Tanuj22 — APPROVED** [review]
Thank You!

**Pull request merged**